appengine/standard/getting-started/test/CloudSqlTest.php

```diff
@@ -20,22 +20,36 @@
 use Google\Cloud\Samples\AppEngine\GettingStarted\CloudSqlDataModel;
 use PHPUnit\Framework\TestCase;
 use PDO;
+use Symfony\Component\Process\Process;
 
 class CloudSqlTest extends TestCase
 {
     use TestTrait;
+    private static $process;
 
     public function setUp(): void
     {
         $connection = $this->requireEnv('CLOUDSQL_CONNECTION_NAME');
         $dbUser = $this->requireEnv('CLOUDSQL_USER');
         $dbPass = $this->requireEnv('CLOUDSQL_PASSWORD');
         $dbName = getenv('CLOUDSQL_DATABASE_NAME') ?: 'bookshelf';
-        $socket = "/tmp/cloudsql/${connection}";
+        $socketDir = '/tmp/cloudsql';
```

Comment:
This should use `DB_SOCKET_DIR`
```suggestion
        $socketDir = $this->requireEnv('DB_SOCKET_DIR');
```